### DISCUSSION TITLE
Lack of Industry Factors in v1.2

### DISCUSSION THREAD

**Discussion by ghost** [USEPA/supply-chain-factors#65] [Q&A] [answered]
V1.2 is only commodity emission factors and not include industry emission factors. Why is that? Will industry emissions factors not be included in all future releases? Is there any guidance as to why the industry factors are so different from the commodity factors?

**Comment by bl-young** [comment, marked as answer]
Correct v1.2 was only released as commodity factors. Industry factors can be generated using the R code provided if you desire.

See also #55 and #42 for other discussions of this topic.

> Is there any guidance as to why the industry factors are so different from the commodity factors?

You may need to clarify this - are you suggesting that in past releases the industry and commodity factors were different? Or in an industry version of the current release? For most sectors, the industry vs commodity factor are not likely to be significantly different. I'll point you to the [original report](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324&Lab=CESER) (as well as Appendix 9) for discussion of industry vs commodity:

**Reply by Brian-Solomon** [reply]
Hello Ben and Wes,
It isn't clear why EPA did not release these industry factors in the most recent release v1.2. 
We are finding it challenging to replicate the instructions and successfully generate the industry factors via R code. We request EPA's assistance here, would it possible for EPA to provide the industry factors directly? We greatly appreciate EPA's continued assistance. Thank you.

**Comment by Brian-Solomon** [comment]
Hi Ben,

Can you provide additional information about this R code to process the current v1.2 into industry-specific factors? This will be crucial for any Category 15 GHG emissions quantification. We are not clear on how to access this code, etc.

Thank you,
-Brian

**Reply by bl-young** [reply]
Yes, you will need to use R and to download the [useeior R package](https://github.com/USEPA/useeior#installation), as well as the code within the [supply-chain-factors repository](https://github.com/USEPA/supply-chain-factors#readme) itself. You can then change the `CommodityorIndustryType` field in the model specification to `Industry` [here](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.1.19-GHG.yml#L10).

Please use the "dev" branch of the supply-chain-factors to avoid an error in generating industry models.

**Reply by WesIngwersen** [reply]
This should work now for code in the main branch

**Reply by jason-frederick** [reply]
Could EPA please post .csv files to Data.gov that contain both the industry and commodity factors?  The files currently posted at https://catalog.data.gov/dataset/supply-chain-greenhouse-gas-emission-factors-v1-3-by-naics-6 contain only industry factors.  I do not have access to or experience with R.
EDIT
I installed R and R Studio.  After knitting, the csv files generated contain only the industry factors, but the yml files at line 10 contain: CommodityorIndustryType: "Commodity".

**Reply by bl-young** [reply]
flagging that I responded to this comment in #88
